extensions/micrometer/runtime/src/main/java/io/quarkus/micrometer/runtime/binder/vertx/VertxHttpServerMetrics.java

```diff
@@ -125,6 +133,7 @@ public void requestReset(HttpRequestMetric requestMetric) {
 
             sample.stop(builder.register(registry));
         }
+        requestMetric.requestEnded();
```

Comment:
Note that the purpose of the path / null check is to avoid recording metrics for a request if it has been ignored.

So, as you're always incrementing (before you know if the path has been ignored), you do always have to decrement.. but I wonder if it is going to be strange having otherwise ignored requests in the active count?

Reply:
Fair enough.